docs/concepts/patching.md

```diff
@@ -67,6 +67,23 @@ from openai import OpenAI
 
 client = instructor.from_openai(OpenAI(), mode=instructor.Mode.JSON)
 ```
+JSON mode is required for [the Gemini Models via OpenAI's SDK](https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/call-gemini-using-openai-library#client-setup).
+
+```python
+import google.auth
+import instructor
+import openai
+creds, project = google.auth.default()
+auth_req = google.auth.transport.requests.Request()
```

Comment:
```suggestion
from openai import OpenAI
from google.auth.transport.requests import Request

creds, project = google.auth.default()
auth_req = Request()
```

Reply:
played around with it this morning. Does the new changes work for you? It's a bit cleaner /more consistent than the `from` and `import`. I'm fine with either, they both work.

Google auth is a bit strange to me, but this definitely works. It's more explicit now so let me know what you think.

Reply:
Yep, that works 👍